packages/vuetify/src/directives/ripple/index.ts

```diff
@@ -148,19 +150,56 @@ function rippleShow (e: MouseEvent | TouchEvent) {
   if (element._ripple.class) {
     value.class = element._ripple.class
   }
-  ripples.show(e, element, value)
+  element._ripple.showRequested = true
+  if (element._ripple.showTimer) {
+    window.clearTimeout(element._ripple.showTimer)
+  }
+  element._ripple.showTimer = window.setTimeout(() => {
+    if (element._ripple && element._ripple.showRequested) {
+      element._ripple.showRequested = false
+      ripples.show(e, element, value)
+    }
+  }, rippleDelayTimeout)
+}
+
+function rippleCancelShow (e: MouseEvent | TouchEvent) {
+  const element = e.currentTarget as HTMLElement | undefined
+  if (!element || !element._ripple) return
+  if (!isTouchEvent(e)) {
+    // It's possible for touch events to fire
+    // as mouse events on Android/iOS, this
+    // will skip the event call if it has
+    // already been registered as touch
+    if (element._ripple.isTouch) return
+  }
+
+  if (element._ripple.showRequested) {
+    element._ripple.showRequested = false
+  } else {
+    rippleHide(e)
+  }
 }
 
 function rippleHide (e: Event) {
   const element = e.currentTarget as HTMLElement | null
```

Comment:
blank line after variable declaration

Reply:
There's an eslint rule for this, we're not using it?

Reply:
Check corresponding coding style doc in Notion. It's not published yet, but I'm still applying its logic in my review.